pkg/azureutils/azure_disk_utils.go

```diff
@@ -525,7 +525,14 @@ func CreateValidDiskName(volumeName string, usedForLabel bool) string {
 }
 
 // GetCloudProviderFromClient get Azure Cloud Provider
-func GetCloudProviderFromClient(kubeClient *clientset.Clientset, secretName, secretNamespace, userAgent string, allowEmptyCloudConfig bool) (*azure.Cloud, error) {
+func GetCloudProviderFromClient(
+	kubeClient *clientset.Clientset,
+	secretName, secretNamespace,
+	userAgent string,
+	allowEmptyCloudConfig bool,
+	enableAzureClientAttachDetachRateLimiter bool,
```

Comment:
We should consider passing `CloudConfiguration` here rather than continuing to add new parameters.